src/main/java/seedu/address/logic/parser/UpdateCommandParser.java

```diff
@@ -60,6 +62,9 @@ public UpdateCommand parse(String args) throws ParseException {
         if (argMultimap.getValue(PREFIX_ADDRESS).isPresent()) {
             editPersonDescriptor.setAddress(ParserUtil.parseAddress(argMultimap.getValue(PREFIX_ADDRESS).get()));
         }
+        if (argMultimap.getValue(PREFIX_DESCRIPTION).isPresent()) {
+            editPersonDescriptor.setDescription(new Description(argMultimap.getValue(PREFIX_DESCRIPTION).get().trim()));
+        }
```

Comment:
Seems to be slightly different from the other implementation.
Not sure if it will affect the program

Reply:
oh yeah probably should slap harder. will change next update
tho wont affect things since description is always valid and nothing to parse